tests/test_workon.py

```diff
@@ -32,4 +32,9 @@ def test_no_symlink(env1):
 def test_no_pew_workon_home(workon_home):
     with temp_environ():
         os.environ['WORKON_HOME'] += '/not_there'
-        assert 'does not exist' in invoke('in', 'doesnt_exist').err
+        assert 'does not exist' in invoke('workon', 'doesnt_exist').err
```

Comment:
I think I preferred `in` instead of `workon` because the former is guaranteed to terminate (if absurdly `workon` would work here, it would leave the test hanging)